charts/ocis/values.yaml

```diff
@@ -194,41 +447,90 @@ hostAliases: []
 # Feature options.
 # Enable or disable features of oCIS.
 features:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   # -- Create demo users on the first startup.
   # Not recommended for production installations.
   demoUsers: false
   # Language related settings
   language:
+    # @schema
+    # type: string
```

Comment:
Shouldn't we use an enum with available languages here?

Reply:
I'm unsure if a exhaustive list about available / supported languages exists.

Even if it exists, we'd need to adapt it with every oCIS release? I'd like to avoid that in the current situation.